src/sentry/conf/server.py

```diff
@@ -399,6 +399,7 @@ def env(
     "sentry.eventstream",
     "sentry.auth.providers.google.apps.Config",
     "sentry.auth.providers.fly.apps.Config",
+    "sentry.auth.providers.saml2.generic.apps.Config",
```

Comment:
For some reason, I needed to add this for templates for SAML2 to be properly resolved.

Reply:
That could be related to the changes in setup.cfg.